README.md

```diff
@@ -322,45 +217,43 @@ OktaAuth.getUser() { response, error in
 Calls the introspection endpoint to inspect the validity of the specified token.
 
 ```swift
-OktaAuth
-  .introspect()
-  .validate(token: token)
-  .then { isActive in
-    print("Is token valid? \(isActive)")
-  }
-  .catch { error in
-    // Error
-  }
+OktaAuth.introspect(token: accessToken, callback: { isValid, error in
+    guard let isValid = isValid else {
+        // Error
+        return
+    }
+    
+    print("Is token valid? \(isValid)")
+})
 ```
 
 ### refresh
 
 Since access tokens are traditionally short-lived, you can refresh expired tokens by exchanging a refresh token for new ones. See the [configuration reference](#configuration-reference) to ensure your app is configured properly for this flow.
 
 ```swift
-OktaAuth
-  .refresh()
-  .then { newAccessToken in
-    print(newAccessToken)
-  }
-  .catch { error in
-    // Error
-  }
+OktaAuth.refresh { newAccessToken, error in
+    if let error = error else {
+        // Error
+        return
+    }
+    
+    // newAccessToken
+}
 ```
 
 ### revoke
 
 Calls the revocation endpoint to revoke the specified token.
 
 ```swift
-OktaAuth
-  .revoke(token: token) { response, error in
-    if error != nil {
-      print("Error: \(error!)")
-    }
-    if let _ = response {
-      print("Token was revoked")
+OktaAuth.revoke(accessToken) { response, error in
+    if let error = error else {
+        // Error
+        return
     }
+    
+    // check response
```

Comment:
Since we're returning `true` or `false` here, maybe update this to:

if (response) {
  // Token was revoked
}

Reply:
Done.